packages/connect-core/src/entities/Organization.ts

```diff
@@ -37,13 +37,21 @@ export default class Organization {
     location: string,
     connector: ConnectorInterface,
     provider: any,
-    network: Networkish
+    network: Network | string
```

Comment:
Ah yes I forgot to make the `Networkish` => `Network` update here. :+1: 

About `string`, do you see a case where this would be needed?

Reply:
I went with the default that Ethers asked for.

Reply:
I think Ethers wants a `Networkish`? https://github.com/ethers-io/ethers.js/blob/ethers-v5-beta/packages/providers/src.ts/web3-provider.ts#L81

But in the case of `Organization`, I think we can restrict it to `Network` to make it easier (since `connect()` is already doing the `Networkish` => `Network` conversion).